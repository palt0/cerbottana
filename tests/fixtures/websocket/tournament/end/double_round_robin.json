{
  "results": [
    [
      "juano ➤"
    ],
    [
      "mammalu"
    ],
    [
      "Pissog",
      "Pippo24/7"
    ],
    [
      "sprinto99"
    ],
    [
      "Carlolucarelli",
      "Barrel of Wine",
      "Foinikorodos"
    ],
    [
      "Alpha",
      "chepokeballs"
    ]
  ],
  "format": "FUTBOLITO BILANCIATO",
  "generator": "Double Round Robin",
  "bracketData": {
    "type": "table",
    "tableHeaders": {
      "cols": [
        "sprinto99",
        "juano ➤",
        "Pissog",
        "Carlolucarelli",
        "Barrel of Wine",
        "Pippo24/7",
        "mammalu",
        "Foinikorodos",
        "Alpha",
        "chepokeballs"
      ],
      "rows": [
        "sprinto99",
        "juano ➤",
        "Pissog",
        "Carlolucarelli",
        "Barrel of Wine",
        "Pippo24/7",
        "mammalu",
        "Foinikorodos",
        "Alpha",
        "chepokeballs"
      ]
    },
    "tableContents": [
      [
        null,
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            1,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            1
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null,
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        null,
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        }
      ],
      [
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "loss",
          "score": [
            0,
            1
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        {
          "state": "finished",
          "result": "win",
          "score": [
            1,
            0
          ]
        },
        null
      ]
    ],
    "scores": [
      9,
      12,
      10,
      8,
      8,
      10,
      11,
      8,
      7,
      7
    ]
  }
}
