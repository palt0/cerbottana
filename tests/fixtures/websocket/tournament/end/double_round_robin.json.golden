[["juano \u27a4"], ["mammalu"], ["Pippo24/7", "Pissog"], ["sprinto99"], ["Barrel of Wine", "Carlolucarelli", "Foinikorodos"], ["Alpha", "chepokeballs"]]